packages/office-ui-fabric-react/src/components/ChoiceGroupOption/ChoiceGroupOption.test.tsx

```diff
@@ -0,0 +1,67 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+import * as renderer from 'react-test-renderer';
+
+import { ChoiceGroupOption } from './ChoiceGroupOption';
+
+describe('ChoiceGroupOption', () => {
+
```

Comment:
I don't know if there's a best practice for authoring snapshot tests, so I just picked a bunch of interesting permutations of the ChoiceGroupOption props.